workflow/envs/vt.yaml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - cryptography=36.0.0
```

Comment:
should we sync version in our docker version which has cryptography=3.4.8

Reply:
done